test/extended/util/db_image_helpers.go

```diff
@@ -157,13 +157,21 @@ func WaitForQueryOutput(oc *CLI, d Database, timeout time.Duration, admin bool,
 		}
 		return false, nil
 	})
+	if err == wait.ErrWaitTimeout {
+		return fmt.Errorf("Timed out waiting for query: %q", query)
```

Comment:
also, why you translate this error messages? why not just return the err itself and get rid of this if